pulsar/consumer_zero_queue_test.go

```diff
@@ -474,6 +474,7 @@ func TestZeroQueueConsumer_Seek(t *testing.T) {
 		Topic:                   topicName,
 		EnableZeroQueueConsumer: true,
 		SubscriptionName:        "sub-1",
+		StartMessageIDInclusive: true,
```

Comment:
irrelevant change again

Reply:
This is necessary, Because this test seeks to N−5 and expects to receive this message next time. 

In fact, StartMessageIDInclusive should also apply to the seek operation.